site/docs/src/json/identity-providers/search-post-request.json

```diff
@@ -0,0 +1,8 @@
+{
+  "search": {
+    "name": "twit",
```

Comment:
Late to the game for functionality and I'm happy to file a tracking, but I think that being able to search by applicationId would be really great. There's no easy way to find out "which identity providers are enabled for this application" other than a lot of pointing and clicking, or downloading all the identity providers and munging the JSON.

Reply:
Let me know if I should file the bug.

Reply:
That change shouldn't be too bad.